docs/rules/a11y-no-generic-link-text.md

```diff
@@ -1,6 +1,8 @@
-# No Generic Link Text [Deprecated]
+# Disallow generic link text (`github/a11y-no-generic-link-text`)
 
-_This rule has been deprecated in favor of `anchor-ambiguous-text` in `eslint-plugin-jsx-a11y`._
+❌ This rule is deprecated. It was replaced by `jsx-a11y/anchor-ambiguous-text`.
```

Comment:
what about this part? is it auto-generated? If so what is it based on?

Reply:
Yeah, based on `meta.deprecated` / `meta.replacedBy`.